execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -727,13 +728,14 @@ fn should_get_first_seigniorage_recipients() {
     );
 
     let mut era_validators: EraValidators = builder.get_era_validators();
-    let snapshot_size = DEFAULT_AUCTION_DELAY as usize + 1;
+    let auction_delay = builder.get_auction_delay();
```

Comment:
We could redefine what `DEFAULT_AUCTION_DELAY` is by forwarding this to the lazily instantiated production config/chainspec.

Reply:
We could do that, however, I prefer using this method as it gives the delay currently being used by the EE itself which I think is a better than having a static that was instantiated once and never gets 'refreshed' or updated.

Reply:
It is changing the test though - it used to use the _default_ value and not it uses the one from latest chainspec. Which may influence the result of the test. Do we want that?

Reply:
Yes, because we want to assert that the functionality of the EE is consistent with values used in production networks